hal/halx86/xbox/reboot.c

```diff
@@ -63,35 +62,47 @@ SMBusWriteByte(UCHAR Address, UCHAR Register, UCHAR Data)
     }
 }
 
+static DECLSPEC_NORETURN
 VOID
-DECLSPEC_NORETURN
-NTAPI
-HalpXboxPowerAction(IN UCHAR Action)
+HalpXboxPowerAction(
+    _In_ UCHAR Action)
 {
+    /* Disable interrupts */
+    _disable();
```

Comment:
@binarymaster @disean I've added this explicit interrupt disable. Please tell me whether this is OK or not, for XBOX.

Reply:
I think this should be fine.